tests/PKSim.R.Tests/IndividualFactorySpecs.cs

```diff
@@ -142,12 +143,115 @@ protected override void Context()
+         _individualCharacteristics = new IndividualCharacteristics
+
+         {
+            Species = CoreConstants.Species.HUMAN,
+            Population = CoreConstants.Population.ICRP,
+            Age = new Parameter
+            {
+               Value = 0.5,
+               Unit = "year(s)",
+            },
+            Weight = new Parameter
+            {
+               Value = 8,
+               Unit = "kg",
+            },
+            Height = new Parameter
+            {
+               Value = 60,
+               Unit = "cm",
+            },
+            Gender = CoreConstants.Gender.Male
+         };
+      }
+
+      protected override void Because()
+      {
+         _individualCharacteristics.AddMoleculeOntogeny(new MoleculeOntogeny {Molecule = "AAA", Ontogeny = "CYP3A4"});
+         _individualCharacteristics.AddMoleculeOntogeny(new MoleculeOntogeny {Molecule = "BBB", Ontogeny = "CYP2E1" });
+         _results = sut.DistributionsFor(_individualCharacteristics);
+      }
+
+      [Observation]
+      public void should_return_all_distributed_parameters_defined_by_the_create_individual_algorithm()
+      {
+         _results.Count().ShouldBeGreaterThan(0);
+         // 2 parameters created 
+         var allCYP3A4Parameters = _results.Where(x => x.ParameterPath.Contains("AAA")).ToList();
+         allCYP3A4Parameters.Count.ShouldBeEqualTo(2);
+         allCYP3A4Parameters[0].ParameterPath.ShouldBeEqualTo(new[]{ "AAA", CoreConstants.Parameters.ONTOGENY_FACTOR }.ToPathString());
+         allCYP3A4Parameters[1].ParameterPath.ShouldBeEqualTo(new[]{ "AAA", CoreConstants.Parameters.ONTOGENY_FACTOR_GI }.ToPathString());
+
+
+         var allCYP2E1Parameters = _results.Where(x => x.ParameterPath.Contains("BBB")).ToList();
+         allCYP2E1Parameters.Count.ShouldBeEqualTo(2);
+         allCYP2E1Parameters[0].ParameterPath.ShouldBeEqualTo(new[] { "BBB", CoreConstants.Parameters.ONTOGENY_FACTOR }.ToPathString());
+         allCYP2E1Parameters[1].ParameterPath.ShouldBeEqualTo(new[] { "BBB", CoreConstants.Parameters.ONTOGENY_FACTOR_GI }.ToPathString());
+
+
+      }
+
+      [Observation]
+      public void should_have_interpolated_the_ontogeny_using_the_expected_values_coming_from_the_database()
+      {
+         // CYP3A4 GI  PMA   Mean Std
+         //             0.77  0.32  1.52
+         //             1.08  0.41  1.45
+         //             1.44  0.5   1.45
+
+
+
+         // CYP3A4 LIV  PMA   Mean Std
+         //             0.77  0.12  2.22
+         //             0.92  0.2   1.92
+         //             1.39  0.49  1.45
+
+         // CYP2E1  PMA   Mean Std
+         //             0.9   0.26  2.51
+         //             1.05  0.6   1.63
+         //             1.12  0.73  1.47
+         //             1.21  0.85  1.39
+         //             1.3   0.92  1.33
+
+
+
+         //0.5 years and 40 weeks GA => PMA is 1.26
+         var allCYP3A4Parameters = _results.Where(x => x.ParameterPath.Contains("AAA")).ToList();
```

Comment:
@Yuri05 I really wanted to add this test to ensure that the values were interpolated as expected. All good here